build_info/openssh.postinst

```diff
@@ -1,4 +1,5 @@
 #!/bin/sh
+[ ! -f /etc/ssh/ssh_host_key ]     && ssh-keygen -q -t rsa1 -f /etc/ssh/ssh_host_key     -N "" -C "" < /dev/null > /dev/null 2> /dev/null
```

Comment:
This postinst shouldn't be needed anymore with that libexec wrapper. Remove it and this should be fine.